NickvisionMoney.GNOME/Resources/org.nickvision.money.css

```diff
@@ -20,8 +20,13 @@
 .account-tag {
   border-radius: 20px;
   font-size: 13px;
-  padding-left: 5px;
-  padding-right: 5px;
+  padding-left: 13px;
+  padding-right: 13px;
```

Comment:
As I understood Tobias asked to decrease paddings to make it look similar to Builder, not increase it.

Reply:
The high amount of "padding" was the width request set to 100

Reply:
Ah, okay. I looked at Builder's tags and they have 12px paddings and ~12.2px font-size. This is very close, so it's fine to leave as it is now. Or you can change, up to you.

Reply:
13px looks fine :)